src/libs/ReportUtils.ts

```diff
@@ -3430,14 +3445,66 @@ function shouldHideReport(report: OnyxEntry<Report>, currentReportId: string): b
     return parentReport?.reportID !== report?.reportID && !isChildReportHasComment;
 }
 
+/**
+ * Checks to see if a report's parentAction is a money request that contains a violation
+ */
+function doesTransactionThreadHaveViolations({
+    report,
```

Comment:
Let's user separate params instead of a single object param

Reply:
this one makes sense because it's only 3 params, so I've put it back.